libs/usb-mocking/README.md

```diff
@@ -0,0 +1,38 @@
+# USB Drive Mocking
+
+The `bash` scripts in this library are useful for mocking USB drives in
+`vxsuite`. The mock consists of a simple loop device backed by an `.img` file.
+The contained filesystem persists and its contents can be separately inspected
+and manipulated.
+
+The current iteration of these scripts only supports one virtual USB drive image
+at a time. You can reset and resize whenever necessary.
+
+### `insert.sh` - Insert Drive
+
+"Inserts" the virtual USB drive. Does not mount the drive.
+
+```bash
+sudo bash insert.sh
```

Comment:
Any way to remove the `sudo bash` part at the beginning?

Reply:
I'm not sure how do do that. Because creating loop devices always require `sudo` permissions, `sudo` has to be somewhere. I can make them executable so instead it's `sudo ./insert.sh`, but that's about all I know how to do at this point.

Reply:
How about wrapper scripts that call `sudo` for you? (I have no idea if this is a good idea or if it's better to explicitly call `sudo`)

Reply:
Ok cool, I would be in favor of a one-time setup personally as long as we feel ok about it from a security perspective. But not a huge deal - mostly just pushing on this to make the interface as easy to use as possible

Reply:
I think it's a good idea but I'm going to punt on it for now. I think, based on other future feedback, we may want a single entry point (single script) which could serve as a wrapper. 

Also, I might be wrong about the `/etc/sudoers` file. Need to think about that. It can allow us to run without a password but not necessarily run without `sudo`. Maybe @eventualbuddha or @amcmanus have a suggestion? In any case, going to punt for now.